src/dom/dom.js

```diff
@@ -2253,6 +2264,9 @@ p5.prototype.createCapture = function(...args) {
     if (domElement.width) {
       videoEl.width = domElement.width;
       videoEl.height = domElement.height;
+      if (flipped) {
+        videoEl.elt.style.transform = 'scaleX(-1)';
```

Comment:
If you call `yourCapture.hide()` and `image(yourCapture, 0, 0)` to draw it to the canvas, is it flipped there too? If not, we may need to update the part of the code where media elements draw to their internal canvas and draw it scaled there too (see this comment for details: https://github.com/processing/p5.js/issues/6441#issuecomment-1747681166)

Reply:
Ya that I missed

Reply:
@davepagurek I did it , now it is working with the capture.hide() and image(capture,0,0);

<img width="457" alt="Screenshot 2024-01-10 232633" src="https://github.com/processing/p5.js/assets/35897449/4dd56f9c-7935-444e-a881-1aa87c6fa66a">
<img width="767" alt="Screenshot 2024-01-10 232926" src="https://github.com/processing/p5.js/assets/35897449/f91ab590-76f6-4bc8-ac17-fb609a2bbc7f">

Reply:
This time i have changed the `_ensureCanvas()`  so that it works fine.

Reply:
Nice, looks great!